tests/fuzzers/les/debug/main.go

```diff
@@ -38,4 +39,12 @@ func main() {
 		os.Exit(1)
 	}
 	les.Fuzz(data)
+	if false {
```

Comment:
The pprof code is still needed? Or should we add a flag to control it?

Reply:
I removed it